services/storage/client-sdk/python/simcore_service_storage_sdk/rest.py

```diff
@@ -71,14 +71,18 @@ def __init__(self, configuration, pools_size=4, maxsize=4):
         )
 
         # https pool manager
+        # We are interested in fast connections, if a connection is established
+        # there is no timeout for file download operations
+        timeout = aiohttp.ClientTimeout(total=None, connect=5, sock_connect=5)
         if configuration.proxy:
             self.pool_manager = aiohttp.ClientSession(
                 connector=connector,
-                proxy=configuration.proxy
+                proxy=configuration.proxy,
+                timeout=timeout,
             )
         else:
             self.pool_manager = aiohttp.ClientSession(
-                connector=connector
+                connector=connector, timeout=timeout
```

Comment:
hmm you know this part of the code is auto-generated right?

Reply:
😲Actually I had no idea. Who/what and generates it? When is it updated?

Reply:
as suggested I've added a test to check that timeouts are in place